tdp/cli/utils.py

```diff
@@ -180,7 +180,6 @@ def rolling_interval(func: FC) -> FC:
 def validate(func: FC) -> FC:
     return click.option(
         "--validate/--no-validate",
-        default=True,
```

Comment:
Put this line in comment and add a TODO ? We will revert it to `True` when the variables validation will work.

Reply:
Yep you're right, that's better.